front/src/applications/operationalStudies/views/Scenario/components/ScenarioHeader.tsx

```diff
@@ -134,6 +134,9 @@ const ScenarioHeader = ({ activeBoards, toggleBoard }: ScenarioHeaderProps) => {
                   onClick={() => {
                     toggleBoard(board);
                   }}
+                  title={
+                    board === 'sdd' || board === 'std' ? t(`boardFullNames.${board}`) : undefined
```

Comment:
```suggestion
                    t(`boardFullNames.${board}`, { defaultValue: undefined })
```

I think it would be easier to maintain, if we maybe want to add other full board names later ?

Reply:
Unfortunately the i18n-checker script trips on this and requires keys for all possible `board` values to be defined.

Opened https://github.com/OpenRailAssociation/osrd/issues/14297 to track this.

Reply:
Ok then LGTM !